newrelic-integration-e2e/internal/docker/docker.go

```diff
@@ -1,17 +1,44 @@
 package docker
 
 import (
+	"fmt"
 	"os"
 	"os/exec"
+	"strings"
 )
 
 const (
 	dockerComposeBin = "docker-compose"
 )
 
-func DockerComposeUp(path string) error{
```

Comment:
my IDE is complaning regarding the dunction starting with the name of the package, should we call it "docker-compose" and the functions run/stop etc?

Reply:
There is also a module https://github.com/compose-spec/compose-go to deal with docker compose... since we need few commands I guess it is better to use directly the shell as you are doing

Reply:
Actually. I searched for a docker-compose SDK but I didn't find it...

Reply:
I renamed  the package  following your suggestions